src/pages/NewSensor/ScanRow.js

```diff
@@ -41,32 +42,39 @@ RectangleButton.propTypes = {
   isSpinning: PropTypes.bool.isRequired,
 };
 
-export const ScanRowComponent = ({ macAddress, blink, isBlinking, isDisabled, selectSensor }) => (
-  <Animatable.View animation="fadeIn" duration={1000} useNativeDriver>
-    <WithFixedDimensions height={60}>
-      <FlexRow flex={0} alignItems="center" justifyContent="flex-end">
-        <RectangleButton
-          isSpinning={isBlinking}
-          isDisabled={isDisabled}
-          onPress={() => blink(macAddress)}
-        />
-        <Spacer space={20} />
-        <TextWithIcon left size="ms" Icon={<WifiIcon />}>
-          {macAddress}
-        </TextWithIcon>
+export const ScanRowComponent = ({ macAddress, blink, isBlinking, isDisabled, selectSensor }) => {
+  const navigation = useNavigation();
+  return (
+    <Animatable.View animation="fadeIn" duration={1000} useNativeDriver>
+      <WithFixedDimensions height={60}>
+        <FlexRow flex={0} alignItems="center" justifyContent="flex-end">
+          <RectangleButton
+            isSpinning={isBlinking}
+            isDisabled={isDisabled}
+            onPress={() => blink(macAddress)}
+          />
+          <Spacer space={20} />
+          <TextWithIcon left size="ms" Icon={<WifiIcon />}>
+            {macAddress}
+          </TextWithIcon>
 
-        <IconButton
-          onPress={() => selectSensor(macAddress)}
-          right
-          labelStyle={localStyles.connectText}
-          label={vaccineStrings.connect}
-          size="ms"
-          Icon={<ChevronRightIcon color={DARKER_GREY} />}
-        />
-      </FlexRow>
-    </WithFixedDimensions>
-  </Animatable.View>
-);
+          <IconButton
+            onPress={() => {
+              // Navigating to the next tab whose name is '1'
+              navigation.navigate('1');
```

Comment:
Other main change is using the navigation ref to navigate here as it's quite a bit quicker. Would like to change the whole `Wizard` component to just use this style rather than the redux-integration that it has